common/config/rush/pnpm-lock.yaml

```diff
@@ -130,12 +130,12 @@ packages:
       integrity: sha512-7C1wodDLnLrdS7rmA/UoItoTAtpZdhkEoaxC7+j5l+LlrcWAe7K2JO1y5psVr5Pe8Y6cUGK4KfpgsAQAcSUDEw==
   /@azure/amqp-common/1.0.0-preview.9:
     dependencies:
-      '@azure/ms-rest-nodeauth': 0.9.3_debug@3.2.7
+      '@azure/ms-rest-nodeauth': 0.9.3_debug@3.2.6
       '@types/async-lock': 1.1.2
       '@types/is-buffer': 2.0.0
-      async-lock: 1.2.8
+      async-lock: 1.3.0
       buffer: 5.7.1
-      debug: 3.2.7
+      debug: 3.2.6
```

Comment:
Debug is being downgraded from 3.2.7 to 3.2.6, and 3.2.6 is deprecated.  We should investigate why this is being downgraded and fix before merging.